src/hooks/useFeedback.ts

```diff
@@ -20,9 +20,13 @@ const useFeedback = () => {
 			submitFeedbackForm: async (category: string, rating: number, text: string, email?: string) => {
 				try {
 					setState({ isSubmitting: true });
-					await feedbackService.submitFeedback(category, rating, text, email);
+					const data = await feedbackService.submitFeedback(category, rating, text, email);
+					if (!data.ok) {
```

Comment:
How does this work? Is `ok` a response field?

Reply:
promise.ok returns a boolean value if 200 was returned. If false, then I throw an error